pages/common/rails-server.md

```diff
@@ -0,0 +1,24 @@
+# rails server
+
+> Serve the Rails app in the current directory using the Puma web server, which comes bundled with Rails.
+> More information: <https://guides.rubyonrails.org/command_line.html#bin-rails-server>.
+
+- Run the web server:
+
+`rails server`
+
+- Run the web server on a specified port:
+
+`rails server --port {{port_number}}`
+
+- Run the web server on a specified IP address:
+
+`rails server --binding {{ip_address}}`
+
+- Run the web server on a specified environment:
+
+`rails server --environment {{environment}}`
```

Comment:
What is the environment here exactly?

Reply:
environment as in `dev`, `production`, etc.